updated cli tutorial (#1003)
fastestimator/fastestimator: tutorial/beginner/t10_cli.md

* [Sending Input Args to `get_estimator`](#t10args)
    * [Using --arg](#t10arg)
    * [Using a JSON file](#t10json)
    * [System argument](#t10system)

<a id='t10intro'></a>
## How Does the CLI Work

> * `fastestimator train`: the command will look for `get_estimator` function then calls the `fit()` method on the returned estimator instance to start the training.
> * `fastestimator test`: the command will look for `get_estimator` function then calls the `test()` method on the returned estimator instance to start the training.
> * `fastestimator run`: the command will look for `fastestimator_run` function and call the function. If `fastestimator_run` is not available, it will then look for `get_estimator` and call `fit()` and/ or `test` dependending on existence of data.

Reviewer: ```suggestion
* `fastestimator train`: the command will look for a `get_estimator` function, invoke it, and then call the `fit()` method on the returned estimator instance to start the training.
* `fastestimator test`: the command will look for a `get_estimator` function, invoke it, and then call the `test()` method on the returned estimator instance to run testing.
* `fastestimator run`: the command will look for a `fastestimator_run` function and invoke it. If `fastestimator_run` is not available, it will instead look for `get_estimator`, invoke it, and then call `fit()` and/ or `test` depending on what data is available within the estimator's Pipeline.
```

``` bash
$ fastestimator train mnist_tf.py --hyperparameters hp.json
```
<a id='t10system'></a>
### System argument

> There are some default system arguments in the `fastestimator train` and `fastestimator test` command, here are a list of them:

Reviewer: ```suggestion
There are some default system arguments in the `fastestimator train` and `fastestimator test` commands. Here are a list of them:
```

* `warmup`:  Only available in `fastestimator train`, it controls whether to perform warmup checking before the actual training starts. Default is True. Users can disable warmup before training by `--warmup False`.

> * `summary`: Available in both `fastestimator train` and `fastestimator test`, this is the same argument used in `estimator.fit()` or `estimator.test()`, it allows users to specify experiment name when generating reports. For example, Users can set experiment name by `--summary exp_name`.

Reviewer: ```suggestion
* `summary`: Available in both `fastestimator train` and `fastestimator test`, this is the same argument used in `estimator.fit()` or `estimator.test()`. It allows users to specify experiment name when generating reports. For example, Users can set experiment name by `--summary exp_name`.
```
